src/main/java/org/myrobotlab/inmoov/Utils.java

```diff
@@ -41,4 +51,57 @@ public static boolean loadFile(String file) {
     return true;
   }
 
+  /**
+   * This method will try to launch a python comman from java land
+   */
+  public static String execPy(String command) {

```

Comment:
just use the python method directly.  No reason to also have this method. This is just extra code that will need to be mainatined.